.github/workflows/ci.yml

```diff
@@ -58,6 +59,34 @@ jobs:
     - run: npm run build
     - run: npm run test
 
+  post_install_test:
+    name: "Test npm postinstall | Node ${{ matrix.node-version }} | ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu, macos, windows]
+        node-version: [15.x, 14.x, 12.x]  # If changing this, also change env.DEFAULT_NODE_VERSION
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          check-latest: true
+      - uses: arduino/setup-protoc@v1
+        with:
+          version: ${{ env.PROTOC_VERSION }}
+          repo-token: "${{ github.token }}"
+      - uses: dart-lang/setup-dart@v1
+        with: {sdk: stable}
+      - run: dart --version
+      - run: npm install
+      - run: rm ./.dev
+      - run: ./node_modules/.bin/ts-node ./tool/prepare-release.ts
+      - run: npm run postinstall
```

Comment:
Should we do a step here to verify that the install is actually functional somehow?

Reply:
It's probably worth it to do a validation here. Maybe a smoke test that runs the binary and compiles a simple stylesheet?

Reply:
Yeah, that sounds good. Doesn't have to be complicated at all, just something that verifies that the library can do anything at all.